src/main/java/org/ice4j/util/ExecutorFactory.java

```diff
@@ -112,4 +112,53 @@ private static ScheduledExecutorService createScheduledExecutor(
         executor.setRemoveOnCancelPolicy(true);
         return Executors.unconfigurableScheduledExecutorService(executor);
     }
+
+
+    /**
+     * Creates a {@link ForkJoinPool} with the given parameters.
+     *
+     * @param parallelism the parallelism level. For default value,
+     * use {@link java.lang.Runtime#availableProcessors}.
+     * @param threadNamePrefix - name prefix for threads created by pool
+     */
+    public static ForkJoinPool createForkJoinPool(
```

Comment:
is this still needed?

Reply:
No, in this pull request - not, though it would be useful in other. For now removing, will add when actually need it.